### ISSUE TITLE
No class today

### ISSUE THREAD

**Issue by thomas--graf** [StonyBrook-Lin637-S20/main#7]
I'm down with a fever. If you have questions about the homework, post them here and I'll try to answer them later today.

**Comment by joannechau** [comment]
Let's say we wrote a list2trie function without reading the hints first and it works. Do we still need to implement an expand function? 

Feel better Thomas!

**Comment by thomas--graf** [comment]
Nope, hints are just hints. There's many ways this can be done, I just outlined one that I think is relatively straightforward. In general, though, I recommend breaking things down into multiple functions that interact with each other, rather than writing a single function that does everything.
